dace/transformation/dataflow/gpu_transform_local_storage.py

```diff
@@ -131,7 +132,10 @@ def match_to_str(self, graph):
         else:
             return str(self.map_entry)
 
-    def apply(self, graph, sdfg):
+    def apply(self, graph: SDFGState, sdfg):
```

Comment:
Pls add the unrelated bugfix to the PR description (searchability)

Reply:
oh dear

Reply:
done